processor/src/main/java/org/mapstruct/ap/internal/model/NestedTargetPropertyMappingHolder.java

```diff
@@ -160,6 +160,15 @@ public NestedTargetPropertyMappingHolder build() {
                         groupedByTP.singleTargetReferences.get( targetProperty )
                     );
 
+                    // If there are multi source parameters that are mapped to the target reference or
```

Comment:
Start active phrasing: , so `We need an update method when: 1) condition 1, 2) condition 2).` I can't understand the current sentence